custom-sidebar-button/README.md

```diff
@@ -22,3 +23,5 @@ the second is the text displayed on the button,
 and the third is an optional boolean to make it open the link in a new tab.
 
 You can have as many buttons as you want, just call the function multiple times.
+
+You can place a line in-between the buttons by calling the first function.
```

Comment:
Please mention the function's name instead of its number in.. what list, actually?

Reply:
I did this because the CreateButton() function is also just referred to as "the function", but ok.